src/schema/README.md

```diff
@@ -9,57 +9,122 @@ Any changes to the specification should be mirrored in the schema.
 
 ## The format of the schema
 
-The schema reflects the files and objects in the specification, as well as
-associations between these objects. Here is a list of the files and subfolders
-of the schema, roughly in order of importance:
-
--   `datatypes/*.yaml`: Data types supported by the specification. Each datatype
-    may support many suffixes. These suffixes are divided into groups based on
-    what extensions and entities are allowed for each. Data types correspond to
-    subfolders (for example, `anat`, `func`) in the BIDS structure.
-
--   `entities.yaml`: A list of entities (key/value pairs in folder and
-    filenames) with associated descriptions and formatting rules. The order of
-    the entities in the file determines the order in which entities must appear
-    in filenames.
-
--   `top_level_files.yaml`: Modality-agnostic files stored at the top level of a
-    BIDS dataset. The schema specifies whether these files are required or
-    optional, as well as acceptable extensions for each.
-
--   `modalities.yaml`: Modalities supported by the specification, along with a
-    list of associated data types. Modalities are not reflected directly in the
-    BIDS structure, but data types are modality-specific.
-
--   `associated_data.yaml`: Folders that are commonly contained within the same
-    folder as a BIDS dataset, but which do not follow the BIDS structure
-    internally, such as `code` or `sourcedata`. The schema specifies which
-    folders are accepted and whether they are required or optional.
-
--   `suffixes/*.yaml`: Suffixes supported by the specification.
-    Each suffix schema file contains, at minimum, `name` and `description`
-    fields. Additionally, it may have a `unit` field defining possible
-    units for data associated with that suffix, as well as fields
-    defining the range or types of values which are allowed for the data,
-    such as `minValue` and `maxValue`.
-
--   `metadata/*.yaml`: Valid fields for sidecar metadata json files.
-    These files contain, at minimum, the following fields: `name`,
-    `description`, and a set of fields for describing the field's data type.
-
-    The data types include `type`, which MUST have a value of
-    `array`, `string`, `integer`, `number`, `object`, or `boolean`.
-    There are additional fields which may define rules that apply to a given
-    type.
-
-    -   `array`: If `type` is `array`, then there MUST be an `items` field at
-        the same level as `type`. `items` describes the data type and rules that
-        apply to the individual items in the array. The same rules that apply to
-        describing data types for the field itself apply to `items`.
-        Additionally, there may be any of the following fields at the same level
-        as `type`: `minItems`, `maxItems`.
-        Here is an example of a field that MUST have three `integer` items:
-        ```yaml
+The schema is divided into two parts: the object definitions and the rules.
```

Comment:
Is this documentation clear enough?

Reply:
I think so. Definitely helped me at least.